### ISSUE TITLE
[ACCESSIBILITY] To-Do list radial button for checking off item (& icon for Editing item) are not labeled properly

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#2039]
**Describe the accessibility issue**
When a non-sighted user is gesturing through app using a screen reader, they are unable to participate in the To do list functionality. 
The radial buttons used as check mark areas for these items need to be labeled clearly as well as once they are checked, the dynamic changes on the screen (showing marked and the new option to clear checked items appears) need to be reported to user somehow without user further gesturing. 

**To Reproduce
Steps to reproduce the behavior:**

Go to '.To Do list area..'
Gesture through to the to do items created

**Expected behavior**
A clear and concise label is given for each item. 

**Actual behavior**
The Checkbox radial button is unlabeled 
The Edit item icon is unlabeled

The screen reader user is hearing a beep sound only when reaching these items. 
Additionally, when the SR user checks an item, the dynamic changes (checked/unchecked) and the appearance of "Clear all checked items" option are not pronounced to the user. 

**Screenshots**
![20220718_104303](https://user-images.githubusercontent.com/82892873/179549407-d0fbe24a-88ca-4360-92b2-2c0f4fc3a720.jpg)
![20220718_103451](https://user-images.githubusercontent.com/82892873/179549451-4427215c-9204-4baf-ba94-db08772f7a0f.jpg)

**Smartphone (please complete the following information):
Device**
iPhone XR- iOS 14.6, Pixel 4a- Android 12

**Additional context**

**Comment by acovher2** [comment]
Found in 4.0.53

**Comment by TodorBachvarov** [comment]
Fixed in 4.1.1

**Comment by acovher2** [comment]
When is 4.1.1 apk being released?

**Comment by acovher2** [comment]
Unable to retest due to Bug for To do list items not saving (#____)

**Comment by acovher2** [comment]
Still Unable to retest due to Bug for To do list items not saving (https://github.com/rokwire/illinois-app/issues/2326)

**Comment by acovher2** [comment]
Verified as fixed in 4.2.34 closing. THank you
